packages/components/affix/__tests__/affix.spec.tsx

```diff
@@ -2,16 +2,14 @@ import { nextTick } from 'vue'
 import { mount } from '@vue/test-utils'
 import { defineGetter, makeScroll } from '@element-plus/test-utils'
 import Affix from '../src/affix.vue'
+import type { VNode } from 'vue'
 
 let clientHeightRestore: () => void
 
-const _mount = (template: string) =>
+const _mount = (render: () => VNode) =>
```

Comment:
We should name this `renderer`

Reply:
I didn't see any special case in your code, can we simply this to:

```ts
const _mount = (template: VNode) => {
  return mount(template, options)
}
```

Reply:
This is a question about code style, we should be consistent, what do you think？@sxzz

Reply:
Different scenario we should write different code, in both cases I am fine with them, but you should avoid writing unnecessary code though, in this case the `setup` is completely redundant to me.

Reply:
Yeah, you are right!